ambry-frontend/src/main/java/com.github.ambry.frontend/GetSignedUrlHandler.java

```diff
@@ -207,12 +207,21 @@ public void onCompletion(Void result, Exception exception) {
           restResponseChannel.setHeader(RestUtils.Headers.DATE, new GregorianCalendar().getTime());
```

Comment:
preProcessRequest for GetSignedUrl will be called in `BlobStorageService.handleGet()`?